app/src/main/java/ar/com/wolox/android/example/ui/viewpager/fragment/ViewPagerFragment.kt

```diff
@@ -27,7 +27,8 @@ class ViewPagerFragment private constructor() : WolmoFragment<FragmentViewpagerB
     override fun handleArguments(arguments: Bundle?) = arguments?.containsKey(FAVOURITE_COLOR_KEY)
 
     override fun init() {
-        binding.viewPager.adapter = SimpleFragmentPagerAdapter(childFragmentManager).apply {
+        binding.viewPager
+        .adapter = SimpleFragmentPagerAdapter(childFragmentManager).apply {
```

Comment:
this indentation seems a little weird and you are not using this file 👀

Reply:
probably a victim of a bunch of viewpager testing 😅